build.sbt

```diff
@@ -1,12 +1,14 @@
+organization := "gatling-mqtt"
+
 name := "gatling-mqtt"
 
-version := "0.0.4-SNAPSHOT"
+version := "0.0.5-SNAPSHOT"
```

Comment:
The version shouldn't be set to "0.0.5-SNAPSHOT" before the version "0.0.4" is released.